packages/framework/src/utils/http.utils.ts

```diff
@@ -1,12 +1,14 @@
 import { checkIsResponseError } from '@novu/shared';
-import { BridgeError, PlatformError } from '../errors';
+import { BridgeError, MissingSecretKeyError, PlatformError } from '../errors';
 
-export const initApiClient = (apiKey: string, baseURL = 'https://api.novu.co') => {
-  const apiUrl = process.env.NOVU_API_URL || baseURL;
+export const initApiClient = (secretKey: string, apiUrl?: string) => {
```

Comment:
It looks like `apiUrl` is always provided as the `resolveApiUrl` function handles it, so I think we can remove the fallback.
```suggestion
export const initApiClient = (secretKey: string, apiUrl: string) => {
```

Reply:
If we do that, we will have to throw an error if the `apiUrl` is missing. Then I thought, do we need the extra bytes for that error?

Reply:
The `resolveApiUrl` provides the fallback here [though](https://github.com/novuhq/novu/pull/6819/files#diff-5e3656182382c7a6da8b4d2e5c7427ffd51d9a0b53a8093310e125547d37dcbaR2), and `initApiClient` always receives an `apiUrl` resolved from that function.

So, an `apiUrl` would always be present and we don't need to handle the undefined `apiUrl` case.

Reply:
You are 100% right. I will update it.